src/cascadia/TerminalSettingsEditor/ColorSchemes.cpp

```diff
@@ -53,9 +56,58 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+        _CurrentBrightColorTable{ single_threaded_observable_vector<Editor::ColorTableEntry>() }
     {
         InitializeComponent();
+
+        const auto foregroundButton{ ForegroundButton() };
+        ToolTipService::SetToolTip(foregroundButton, box_value(RS_(L"ColorScheme_Foreground/Text")));
+        Automation::AutomationProperties::SetName(foregroundButton, RS_(L"ColorScheme_Foreground/Text"));
+
+        const auto backgroundButton{ BackgroundButton() };
+        ToolTipService::SetToolTip(backgroundButton, box_value(RS_(L"ColorScheme_Background/Text")));
+        Automation::AutomationProperties::SetName(backgroundButton, RS_(L"ColorScheme_Background/Text"));
+
+        const auto cursorColorButton{ CursorColorButton() };
+        ToolTipService::SetToolTip(cursorColorButton, box_value(RS_(L"ColorScheme_CursorColor/Text")));
+        Automation::AutomationProperties::SetName(cursorColorButton, RS_(L"ColorScheme_CursorColor/Text"));
+
+        const auto selectionBackgroundButton{ SelectionBackgroundButton() };
+        ToolTipService::SetToolTip(selectionBackgroundButton, box_value(RS_(L"ColorScheme_SelectionBackground/Text")));
+        Automation::AutomationProperties::SetName(selectionBackgroundButton, RS_(L"ColorScheme_SelectionBackground/Text"));
+
+        const auto comboBox{ ColorSchemeComboBox() };
+        Automation::AutomationProperties::SetName(comboBox, RS_(L"ColorScheme_Name/Header"));
+        Automation::AutomationProperties::SetFullDescription(comboBox, RS_(L"ColorScheme_Name/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip"));
+        ToolTipService::SetToolTip(comboBox, box_value(RS_(L"ColorScheme_Name/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip")));
+
+        Automation::AutomationProperties::SetName(RenameButton(), RS_(L"Rename/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip"));
+
+        const auto nameBox{ NameBox() };
+        Automation::AutomationProperties::SetName(nameBox, RS_(L"ColorScheme_Name/Header"));
+        Automation::AutomationProperties::SetFullDescription(nameBox, RS_(L"ColorScheme_Name/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip"));
+        ToolTipService::SetToolTip(nameBox, box_value(RS_(L"ColorScheme_Name/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip")));
+
+        Automation::AutomationProperties::SetName(RenameAcceptButton(), RS_(L"RenameAccept/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip"));
+        Automation::AutomationProperties::SetName(RenameCancelButton(), RS_(L"RenameCancel/[using:Windows.UI.Xaml.Controls]ToolTipService/ToolTip"));
+        Automation::AutomationProperties::SetName(AddNewButton(), RS_(L"ColorScheme_AddNewButton/Text"));
+        Automation::AutomationProperties::SetName(DeleteButton(), RS_(L"ColorScheme_DeleteButton/Text"));
+    }
+
+    void ColorSchemes::SizeChanged(IInspectable const& /*sender*/, Windows::UI::Xaml::SizeChangedEventArgs const& e)
```

Comment:
sorry, we couldn't use a VSM because what now? we would have to retemplate the entire thing? heck naw

Reply:
The VSM honestly just made the code more complex to change this one value. The changes would look something like this...
- add the VSM; define 2 states (compact and standard)
- Either...
  - set up an AdaptiveTrigger to switch between the states (I'd probably have to bind that to the `_colorPanelHorizontalWidth + indentMargin`, which would be annoying to set up in the code)
  - set up `SizeChanged` (like we do here), and call `VisualStateManager::GoToState()`

Reference: https://docs.microsoft.com/en-us/windows/uwp/design/layout/layouts-with-xaml#adaptive-layouts-with-visual-states-and-state-triggers